### DISCUSSION TITLE
Template for TADx

### DISCUSSION THREAD

**Discussion by philippart-s** [lyonjs/shortvid.io#621] [Conference / Meetup Communication]
Hi,

I've tried to make my own template for my meetup TADx but it's to complex for me 😅.
What I want to do : juste use the default template _talk branded_ and add a background custom image (animated or not).

How do you want to proceed?

Thanks 😊

**Comment by Slashgear** [comment]
Hi @philippart-s 

Do you have some assets that you want to use ?

With @CruuzAzul we could try to work on something

**Reply by mickaelalvs** [reply]
Hi! I'd been talking to Fanny K. about making a custom template for TADx, maybe that'll be better 😊

**Reply by philippart-s** [reply]
Sorry for the delay the come back from DevFest Nantes was busy 😉.
As you wish (use existing assets or create new one).
I add our existing assets.


![Banniere-Twitter-TADx](https://github.com/lyonjs/shortvid.io/assets/26224751/2b45fdc9-46d4-4e2f-a009-31764aeafbc6)

![TADx_HD](https://github.com/lyonjs/shortvid.io/assets/26224751/4629de55-bd12-40f1-afad-f03e379264bb)

**Reply by mickaelalvs** [reply]
@philippart-s Do you have TADx visual elements in SVG format? It might be easier to animate 😊

**Reply by philippart-s** [reply]
hi, unfortunately no :(
I've try some online tools to convert the pictures but none of them generate a good SVN file
